README.md

```diff
@@ -74,18 +74,24 @@ Charactersheet uses HTML Local Storage to ensure that all your data is stored lo
 Installation
 ============
 
-Installing charactersheet is pretty simple. 
+Installing charactersheet is pretty simple.
 
         mkdir charactersheet
         git clone https://github.com/charactersheet/charactersheet.git charactersheet
-        cd charactersheet 
-        npm install 
+        cd charactersheet
+        npm install
         npm test
 
 That's it! If everything is green, then you're good to go!
 
 
+Have questions or just want to chat?
+=====================
+
+Join the conversations at: https://gitter.im/Sonictherocketman/adventurerscodex
```

Comment:
What do you think about using the Gitter Badge at the top instead (or both)?

[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/Sonictherocketman/adventurerscodex)

Markdown Source:
`[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/Sonictherocketman/adventurerscodex)`